src/locale/en.ts

```diff
@@ -25,6 +25,8 @@ export const en = {
   detailPanelInScheme: "In vocabulary",
   detailPanelDefinition: "Definition",
   detailPanelStereotype: "Stereotype",
+  setCanvasLanguage: "Workspace language",
```

Comment:
Do you think these names for the individual settings are OK? I thought about it a bit but haven't been able to come up with better ones.

Reply:
What about "Default vocabulary language" or "Displayed vocabulary language"?